cmd/gpud/command/update.go

```diff
@@ -83,3 +82,20 @@ func cmdUpdateCheck(cliContext *cli.Context) error {
 	fmt.Printf("latest version: %s\n", ver)
 	return nil
 }
+
+func fetchVersion(track string) (string, error) {
```

Comment:
I re-read the update-related codes and I feel that my previous suggestions were not very good, I'm sorry for wasting your time.
I have two questions:
1. Why do we have a situation like `v0.4.3\n`? Do we have a bug somewhere else that causes this problem?
2. If Trim is required, can it be modified in `detectLatestVersion`? It should be enough to just trim the fetchedVer variable once.

Reply:
> Why do we have a situation like v0.4.2\n

Sometimes, release owner uploaded the unstable_latest.txt file with trailing whitespaces (as it's not fully automated yet).

> can it be modified in detectLatestVersion

Let me check